frontend/services/src/maps/mapsService.ts

```diff
@@ -0,0 +1,66 @@
+import { BasicData } from "@/components/events/create/forms/BasicForm";
+import { Environment, getEnvironment } from "@/utilities/environment";
+import { useLoadScript } from "@react-google-maps/api";
+
+const libraries: "places"[] = ["places"];
+
+const getGoogleMapsApiKey = () => {
+  const env = getEnvironment();
+  if (env === Environment.DEVELOPMENT) {
```

Comment:
we need preview in this as well.. probably the conditional is IF env == PROD and IF env != PROD, rather than checking for dev